src/udev/udev-builtin-net_id.c

```diff
@@ -600,6 +603,93 @@ static int names_platform(sd_device *dev, NetNames *names, bool test) {
         return 0;
 }
 
+#define DEVICETREE_ALIASES "/proc/device-tree/aliases"
+
+static int dev_devicetree_onboard(sd_device *dev, NetNames *names) {
+        const char *alias, *ofnode_path, *ofnode_syspath, *parent_syspath;
+        _cleanup_free_ char *devicetree_syspath = NULL;
+        sd_device *aliases, *ofnode, *parent;
+        int r;
+        unsigned index;
```

Comment:
given libc has a funciton called "index" maybe pick a different name. old gcc used to warn about cases like this even

Reply:
Huh. I'll shorten it to `i`.